src/index.js

```diff
@@ -372,4 +388,22 @@ export default class KintoClient {
     return this.execute(requests.deleteBucket(_bucket, reqOptions))
       .then(res => res.json);
   }
+
+  /**
+   * Deletes all buckets on the server.
+   *
+   * @ignore
+   * @param  {Object}  options         The options object.
+   * @param  {Boolean} options.safe    The safe option.
+   * @param  {Object}  options.headers The headers object option.
+   * @return {Promise<Object, Error>}
+   */
+  deleteBuckets(options={}) {
+    return this.ensureSupported("1.4", "2.0")
```

Comment:
> Is this seriously a big issue? We can always revisit the max version at the time this next major of the protocol is out...

After thinking about it, yes, this is it! We can bump the next supported major whenever :) 

Sorry, I replied to this comment without going very deeply :( I had no intention to confuse you!